Update Notification

EkinOox opened the pull request:
Notifications Achieved

EkinOox committed:
Envoie depuis controller (notifications)

controller et non client

mathis-lambert committed:
Merge branch 'kyllian' of github.com:BUT-MMI3/VisioConf into kyllian

EkinOox committed:
Notification Update

EkinOox committed:
Update NoyauAccueil.jsx

EkinOox committed:
Changing  message's names

EkinOox committed:
Update Notification

Notification Achieved

EkinOox committed:
Merge branch 'main' into kyllian

mathis-lambert commented on api-express/src/classes/Messages.js:
```diff
@@ -95,6 +94,14 @@ class Messages {
                         },
                         id: member.user_socket_id
                     });
+                    this.controller.send(this, {
```
on a dit que la structure d'une notification c'est : 
```json
{
  "type": 'Alert' ou 'Warning' ou 'Info',
  "content": "Message de la notification,
  "data": "optionnel mais tu peux mettre des données supplémentaires ici si besoin"
}
```

EkinOox committed:
Changement Notifications

Changement de la structure des notifications ainsi que l'ajout d'une redirection vers la discussion au click de la notif

mathis-lambert commented on api-express/src/classes/Messages.js:
```diff
@@ -95,6 +94,18 @@ class Messages {
+                    this.controller.send(this, {
+                        demande_notification: {
+                            type: "Info",
+                            content: lastMessage,
```
le "content" de la notification est obligatoirement un string, si tu veux lastMessage met le dans data
@EkinOox

mathis-lambert committed:
Merge branch 'kyllian' of github.com:BUT-MMI3/VisioConf into kyllian

mathis-lambert committed:
[FIX] Fixed data type of Notification
